dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/startup/SchedulerStart.java

```diff
@@ -130,59 +157,98 @@ public void execute( )
                 jobs.append( unexecutedJob ).append( "\n" );
             }
 
-            messageService.sendSystemErrorNotification( "Scheduler startup", new Exception( "Scheduler started with one or more unexecuted jobs:\n" + jobs ) );
+            messageService.sendSystemErrorNotification( "Scheduler startup",
+                new Exception( "Scheduler started with one or more unexecuted jobs:\n" + jobs ) );
         }
     }
 
-    private void addDefaultJobs( List<JobConfiguration> jobConfigurations)
+    private void addDefaultJobs( List<JobConfiguration> jobConfigurations )
     {
         log.info( "Setting up default jobs." );
-        if ( addDefaultJob( DEFAULT_FILE_RESOURCE_CLEANUP, jobConfigurations ) )
+        if ( verifyNoJobExist( DEFAULT_FILE_RESOURCE_CLEANUP, jobConfigurations ) )
         {
             JobConfiguration fileResourceCleanUp = new JobConfiguration( DEFAULT_FILE_RESOURCE_CLEANUP,
                 FILE_RESOURCE_CLEANUP, CRON_DAILY_2AM, null, false, true );
             addAndScheduleJob( fileResourceCleanUp );
         }
 
-        if ( addDefaultJob( DEFAULT_DATA_STATISTICS, jobConfigurations ) )
+        if ( verifyNoJobExist( DEFAULT_DATA_STATISTICS, jobConfigurations ) )
         {
             JobConfiguration dataStatistics = new JobConfiguration( DEFAULT_DATA_STATISTICS, DATA_STATISTICS,
                 CRON_DAILY_2AM, null, false, true );
-            SchedulerUpgrade.portJob( systemSettingManager, dataStatistics,"lastSuccessfulDataStatistics" );
+            SchedulerUpgrade.portJob( systemSettingManager, dataStatistics, "lastSuccessfulDataStatistics" );
 
             addAndScheduleJob( dataStatistics );
         }
 
-        if ( addDefaultJob( DEFAULT_VALIDATION_RESULTS_NOTIFICATION, jobConfigurations ) )
+        if ( verifyNoJobExist( DEFAULT_VALIDATION_RESULTS_NOTIFICATION, jobConfigurations ) )
         {
-            JobConfiguration validationResultNotification = new JobConfiguration( DEFAULT_VALIDATION_RESULTS_NOTIFICATION,
-                VALIDATION_RESULTS_NOTIFICATION, CRON_DAILY_7AM, null, false, true );
+            JobConfiguration validationResultNotification = new JobConfiguration(
+                DEFAULT_VALIDATION_RESULTS_NOTIFICATION, VALIDATION_RESULTS_NOTIFICATION, CRON_DAILY_7AM, null, false,
```

Comment:
Codestyle

Reply:
The Dhis Code Formatter for eclipse formats it in this way. I will change it back to how it was then.